Popular Movies/Popular Movies/Controllers/RssFeedTableViewController.h

```diff
@@ -8,6 +8,5 @@
 
 #import <UIKit/UIKit.h>
 
```

Comment:
**Same suggestion:** If you inherit **UITableViewController** class it is not needed to implement **UITableViewDelegate** and **UITableViewDataSource** protocols. This is needed when you inherit **UIViewController** class.